### PR TITLE
Resolved #726

### PR DESCRIPTION
**Pull request by aripddev**

### CODE REVIEW

File: src/main/resources/META-INF/resources/primefaces-extensions/letteravatar/0-letteravatar.js [hunk 1]
```diff
@@ -30,7 +30,8 @@
         canvas.height = size;
         context = canvas.getContext("2d");
 
-        context.fillStyle = colours[colourIndex - 1];
+//        context.fillStyle = colours[colourIndex - 1];
```
Comment (jepsar) [inline src/main/resources/META-INF/resources/primefaces-extensions/letteravatar/0-letteravatar.js]: I'd simply remove this line

**Review by melloware — APPROVED** [review]
Nice PR!  is it safe to merge or do you have any more changes?

**Comment by aripddev** [comment]
Not for this issue but I can add an improvement for more than two names, like "Johann Chrysostom Wolfgang Amadeus Mozart" as "JM". Do I need a new issue?

**Comment by melloware** [comment]
Create a new issue but you can reuse this PR.  I won't merge until you give me the go ahead.  but lets track the new issue.

**Comment by aripddev** [comment]
Also need to add `title` and `alt` attributes to show full text when mouse over

**Comment by aripddev** [comment]
@melloware 
All completed. You can merge it if no problem.

**Pull request merged**